__tests__/build-newsletter-edition.test.js

```diff
@@ -0,0 +1,19 @@
+import { saveNewsletterEditions } from '/Users/pilardeharo/TNC_Code/next-tinynewsdemo/script/build-newsletter-editions.js';
```

Comment:
We can delete this since it doesn't have a real test case. I think this was leftover from our pair programming.